### ISSUE TITLE
WCAG 3.1.1: Ensures every HTML document has a lang attribute (html)

### ISSUE THREAD

**Issue by roelvangils** [11ways/insights-tests#1]
#### Issue
`Ensures every HTML document has a lang attribute` ([html-has-lang](https://accessibilityinsights.io/info-examples/web/html-has-lang))
#### Target application
[De Belgische Kamer van volksvertegenwoordigers](https://www.dekamer.be/kvvcr/index.cfm)
#### Element path
html
#### Snippet
`<html xmlns="http://www.w3.org/1999/xhtml" stopthemadness="banners beacon drag select shadow" data-lt-installed="true">`
#### How to fix
    Fix any of the following:
      The <html> element does not have a lang attribute
#### Environment
Chrome version 111.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
